pkg/controllers/management/auth/crtb_handler.go

```diff
@@ -15,16 +16,17 @@ import (
 )
 
 const (
-	clusterResource             = "clusters"
-	membershipBindingOwner      = "memberhsip-binding-owner"
-	membershipBindingOwnerIndex = "auth.management.cattle.io/membership-binding-owner"
-	crtbInProjectBindingOwner   = "crtb-in-project-binding-owner"
-	prtbInClusterBindingOwner   = "prtb-in-cluster-binding-owner"
-	rbByOwnerIndex              = "auth.management.cattle.io/rb-by-owner"
-	rbByRoleAndSubjectIndex     = "auth.management.cattle.io/crb-by-role-and-subject"
-	ctrbMGMTController          = "mgmt-auth-crtb-controller"
-	rtbLabelUpdated             = "auth.management.cattle.io/rtb-label-updated"
-	rtbCrbRbLabelsUpdated       = "auth.management.cattle.io/crb-rb-labels-updated"
+	clusterResource              = "clusters"
+	membershipBindingOwnerLegacy = "memberhsip-binding-owner"
+	membershipBindingOwner       = "membership-binding-owner"
```

Comment:
Can you add comments here to clarify the two labels and call out that they are indeed different